### ISSUE TITLE
Example / walkthrough of DPNN + DASO usage

### ISSUE THREAD

**Issue by coquelin77** [helmholtz-analytics/heat#741]
Walk through of using DPNNs with DASO or without. 

needs a dataloader, optimizer, network, loss, train loop, val loop

**Comment by bhagemeier** [comment]
Daniel, I removed the so far empty tutorial for DPNN, which I think would have been the goal of this ticket. I have created a [branch](/helmholtz-analytics/heat/tree/features/741-dpnn-walkthrough) of the documentation branch right before the removal. This way, it will be a tad easier to copy the required change in tutorials.rst from there. You could as well just revert commit dc61b72b4cbcf956f955df3974be3e5b6192b1fb to re-add the tutorial to the structure once it's fully available.

**Comment by ClaudiaComito** [comment]
Is this solved by our [nn examples](https://github.com/helmholtz-analytics/heat/tree/main/examples/nn)?

**Comment by mrfh92** [comment]
Stil open and would be useful to have, although not high priority at the moment

Reviewed within #1109.
